ThunkLibs/HostLibs/CMakeLists.txt

```diff
@@ -10,6 +10,12 @@ if (ENABLE_CLANG_THUNKS)
   add_link_options(${LD_OVERRIDE})
 endif()
 
+find_program(CCACHE_PROGRAM ccache)
```

Comment:
Does HostLibs need this? Host thunks are compiled using the host toolchain and hence follow the setup from the main `CMakeLists.txt`, which already sets up ccache AFAICT.

Reply:
You're right, removed.